doc/whats_new/v1.0.rst

```diff
@@ -20,6 +20,8 @@ Changelog
   :class:`decomposition.MiniBatchDictionaryLearning`, :class:`decomposition.SparsePCA`
   and :class:`decomposition.MiniBatchSparsePCA` to be convex and match the referenced
   article. :pr:`19210` by :user:`Jérémie du Boisberranger <jeremiedbb>`.
+- |Fix| :func:`utils.estimator_html_repr` now escapes all the estimator
+  descriptions in the generated HTML. :pr:`21493` by `Aurélien Geron`_.
```

Comment:
The changelog is organized by header and your name requires `:user:` to link your name to your github username:

```suggestion

:mod:`sklearn.utils`
....................

- |Fix| :func:`utils.estimator_html_repr` now escapes all the estimator
  descriptions in the generated HTML. :pr:`21493` by
  :user:`Aurélien Geron <ageron>`.
```